.github/workflows/klocwork-pull-request.yml

```diff
@@ -62,12 +62,20 @@ jobs:
         run: kwinject --output kwinject.out ninja -C build -v -k0
 
       - name: fully analyze build of base ref
-        run: kwcheck run --build-spec kwinject.out -F scriptable --report issues.txt --project-dir /home/klocwork/.kwlp --settings-dir /home/klocwork/.kwps
+        run: kwcheck run --build-spec kwinject.out -F scriptable --report /home/klocwork/parent_issues.txt --project-dir /home/klocwork/.kwlp --settings-dir /home/klocwork/.kwps
 
-      - name: ignore issues of base ref
-        run: |
-          count=$(wc -l < issues.txt)
-          test "$count" = "0" || kwcheck set-status 1-"$count" --status ignore --project-dir /home/klocwork/.kwlp
+      - name: get parent issue ids
+        run: awk -F ';' '{print $1}' /home/klocwork/parent_issues.txt | sort > /home/klocwork/parent_issues_ids.txt
```

Comment:
Actually your initial solution using non-numeric `sort` was better, since that way you can make use of the (default) ordering check in `comm`. So I would suggest you revert to non-numeric `sort`, but add a comment **why** the sort is non-numeric.

Reply:
I have added the comment and changed back to default sort.